binding/android/README.md

```diff
@@ -106,6 +164,22 @@ OrcaSynthesizeParams params = new OrcaSynthesizeParams.Builder()
 - `setSpeechRate()`: Controls the speed of the generated speech. Valid values are within [0.7, 1.3]. A higher value
   produces speech that is faster. The default is `1.0`.
 
+### Alignment Metadata
+
+Along with the raw PCM or saved audio file, Orca returns metadata for the synthesized audio in single synthesis mode.
+The `Orca.WordAlignment` object has the following properties:
```

Comment:
Same as for web `Orca.WordAlignment` needs to be changed to the class name in java.